src/OpenColorIO/Platform.h

```diff
@@ -124,43 +124,43 @@ OCIO_NAMESPACE_ENTER
 //       In the meantime, hardcode to x86
 #define OCIO_LITTLE_ENDIAN 1  // This is correct on x86
 
-    /*
-     * Mutex classes
-     */
+/*
+ * Mutex classes
+ */
 
 #ifdef WINDOWS
 
-    class _Mutex {
-    public:
+class _Mutex {
```

Comment:
Sorry for the late comment, and maybe this has been covered already...

But now that OCIO is C++11 compliant, shouldn't this be excised in favor of std::mutex?

Reply:
opened issue #583

Reply:
+1